File: pgm/ABC_Java/ABC_236_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] S = reader.readLine().split ("");
        final String[] arr = reader.readLine().split (" ");
        final int a = Integer.parseInt (arr[0]);
        final int b = Integer.parseInt (arr[1]);
        String s = S[a - 1];
        S[a - 1] = S[b - 1];
        S[b - 1] = s;
        System.out.println (String.join ("", S));
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_236_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val blank: String = ""
const val space: String = " "
const val one: Int = 1

fun main() {
    val S: CharArray = readln().toCharArray()
    val (a: Int, b: Int) = readln().split(space).map { it.toInt() }
    val keep = S[a - one]
    S[a - one] = S[b - one]
    S[b - one] = keep
    println(S.joinToString(blank))
    kotlin.system.exitProcess(0)
}